src/poisoning.md

```diff
@@ -1,35 +1,74 @@
 # Poisoning
```

Comment:
can you drop changes in this file from this PR?
I think you have another PR for poisoning.

Reply:
Ok thanks. I'm not super familiar with how PR in github works either.